paper/paper.bib

```diff
@@ -8,20 +8,17 @@
 
 @misc{manif,
 	Author = {Jeremie Deray and Joan Sol\`a},
-	Date-Added = {2019-01-09 19:46:45 +0100},
-	Date-Modified = {2019-01-09 22:35:53 +0100},
-	Howpublished = {\url{https://github.com/artivis/manif}},
+	url = {https://github.com/artivis/manif},
 	Month = {January},
 	Read = {0},
 	Title = {\texttt{manif}: a small {C++} header-only library for {L}ie theory.},
 	Year = {2019},
-	Bdsk-Url-1 = {github.com:artivis/manif.git},}
 
 @misc{Sophus,
 	Author = {Hauke Strasdat},
 	Title = {Sophus},
 	year = 2018,
-	howpublished = {[online] \url{https://github.com/strasdat/Sophus}},
+	url = {https://github.com/strasdat/Sophus},
 	Bdsk-Url-1 = {https://github.com/strasdat/Sophus},}
```

Comment:
This tag was removed for other entries. Is it problematic?

Reply:
Yes, howpublished is problematic - simply using url is not

Reply:
I meant, is `Bdsk-Url-1` tag problematic? You have removed it in other bib entries but not here.

Reply:
sorry.  It's not really problematic, but anything extra right now seems to be confusing our system and getting into the XML data.  I'll remove it.

Reply:
ok, now removed